main.js

```diff
@@ -1 +1,64 @@
 import customers from './customers.js'
+
+//make first letter capital
+function capital(string){
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function listCustomers(personArray) {
+  let customerList = document.querySelector("#customer-list");
+  //console.log(customerList);
+
+  for (let customer of customers) {
```

Comment:
This is a good start. You'll want to generalize, loop over `personArray` here, and then specify which array when you call the function. `listCustomers(customers)`

Reply:
okay. Thank you